styles/responsivelLandscape320px.css

```diff
@@ -0,0 +1,181 @@
+@media screen and (orientation: landscape) and (min-width:568px) and (min-height:320px) {
+ 
+    .header_logo__style{
+        position: relative;                
+        left: 8%;
+    }
+
+    .menu_checkbox__style{
+        position: relative;
+        left: 22%;         
+    }
+
+    .scroll_container__style{
+        position: fixed;
+        top: 87%;        
+    }
+
+    .fresh_life_container__style{
+        height: 420px;
+         
+    }
+
+    .about_us_container__style{
+        height: 422px;
+    }
+
+    .about_us_content__style{
+        background-position: -300% 35%;
+        position: relative;
+        top: -24px;          
+    }
+
+    .about_us_content-h2__style{    
+        width: 24%; 
+        position: relative;   
+        top: 35px;
+        left: 60%;
+    }
+    
+    .about_us_content-p__style{
```

Comment:
Same for this selector - poorly given class name and common rules are in other `media`s for this selector